src/McBopomofo.cpp

```diff
@@ -750,6 +750,7 @@ void McBopomofoEngine::keyEvent(const fcitx::InputMethodEntry& /*unused*/,
       return;
     }
 
+
```

Comment:
Can we remove this empty line? `McBopomofo.cpp` shouldn't have any diff at all based on what this PR does.